yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -0,0 +1,264 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/datafile"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/dbzm"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/metadb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+var liveMigrationReportCmd = &cobra.Command{
+	Use:   "report",
+	Short: "This command will print the report of any live migration workflow.",
+	Long:  `This command will print the report of the live migration or live migration with fall-forward or live migration with fall-back.`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		validateExportDirFlag()
+		migrationStatus, err := metaDB.GetMigrationStatusRecord()
+		if err != nil {
+			utils.ErrExit("error while getting migration status: %w\n", err)
+		}
+		streamChanges, err := checkWithStreamingMode()
+		if err != nil {
+			utils.ErrExit("error while checking streaming mode: %w\n", err)
+		}
+		migrationUUID, err = uuid.Parse(migrationStatus.MigrationUUID)
+		if err != nil {
+			utils.ErrExit("error while parsing migration UUID: %w\n", err)
+		}
+		if streamChanges {
+			getTargetPassword(cmd)
+			migrationStatus.TargetDBConf.Password = tconf.Password
+			if migrationStatus.FallForwardEnabled {
+				getFallForwardDBPassword(cmd)
+				migrationStatus.FallForwardDBConf.Password = tconf.Password
+			}
+			if migrationStatus.FallbackEnabled {
+				getSourceDBPassword(cmd)
+				migrationStatus.SourceDBAsTargetConf.Password = source.Password
+			}
+			liveMigrationStatusCmdFn(migrationStatus)
+		} else {
+			utils.ErrExit("export-type 'snapshot-and-changes' is not enabled for this migration")
+		}
+	},
+}
+
+type rowData struct {
+	TableName        string
+	DBType           string
+	SnapshotRowCount int64
```

Comment:
I wonder if we should have two fields "Exported SNapshot" and "Imported Snapshot". 
That would be consistent with the inserts/updates/deletes and also make it more clear that the snapshot is exporteed from the source and imported into target(and ff)

Reply:
Hmm...you are suggesting having an 'Exported snapshot', and an 'Imported snapshot' separate fields but then we never export snapshots from DB other than `Source`. So for that case, it will be 0 0.
I think that it should be implicit that the Snapshot row count for Source is exported and Target will be imported only but for inserts/updates/deletes this can be the other way around as well. so having explicit Exported/Imported mention makes sense there.

Reply:
Created a ticket - https://yugabyte.atlassian.net/browse/DB-8658, will do this change in the separate PR.